concepts/Common/Shadow DOM/Shadow DOM.md

```diff
@@ -0,0 +1,23 @@
+[Shadow DOM](https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_shadow_DOM) allows you to attach a DOM subtree to an element and have the internals of this tree hidden from JavaScript and CSS running in the page.
+
+#include common-ctp-note-wo-devextreme
+
+At present, DevExtreme components only partly support Shadow DOM. The following limitations apply:
+
+- Diagram and Gantt do not support Shadow DOM.
+
+- CSS selectors in properties like [target](/Documentation/ApiReference/UI_Components/dxContextMenu/Configuration/#target) or [container](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#container) are not supported. Pass a DOM element instead.
+
+- Popup, Tooltip, Popover, Toast, and LoadPanel render the dialog window in the `<body>` element, not in Shadow DOM.
+
+- DevExtreme stylesheets must be included in the Light DOM for Shadow DOM components to render correctly.
```

Comment:
Just adding a stop for clarity. I hope I didn't mess it up.  

```suggestion
- DevExtreme stylesheets must be included in the Light DOM. Otherwise Shadow DOM components may render correctly.
```